media/js/zamboni/files.js

```diff
@@ -1,4 +1,5 @@
-"use strict";
```

Comment:
This will make every file concatentated with this strict. We might not want that.